docs/docusaurus/docs/guides/expectations/how_to_edit_an_existing_expectationsuite.md

```diff
@@ -0,0 +1,160 @@
+
+</details>
+
+<details>
+<summary>
+
+## Steps
+
+### 1. Import the Great Expectations module and instantiate a Data Context
+
+The simplest way to create a new Data Context is by using the `get_context()` method.
+
+```python name="tests/integration/docusaurus/expectations/how_to_edit_an_expectation_suite get_context"
+```
+
+### 2. Create a Validator from Data 
+
+Run the following command to connect to `.csv` data stored in the `great_expectations` GitHub repository:
+
+```python name="tests/integration/docusaurus/expectations/how_to_edit_an_expectation_suite create_validator"
+```
+
+### 3. Create Expectations with Validator 
+
+Run the following commands to create two Expectations. The first Expectation uses domain knowledge (the `pickup_datetime` shouldn't be null), and the second Expectation uses `auto=True` to detect a range of values in the `passenger_count` column.
+
+
+```python name="tests/integration/docusaurus/expectations/how_to_edit_an_expectation_suite add_2_expectations"
+```
+
+Under the hood, the Validator will be creating and updating an Expectation Suite, which we can view next. 
+
+### 4. View the Expectations in the Expectation Suite
+
+There are a number of different ways that this can be done, with one way being using the `show_expectations_by_expectation_type()` function, which will use `prettyprint` to print the Suite to the console in a way that can be easily visualized. 
+
+First load the `ExpectationSuite` from the `Validator`: 
+
+```python name="tests/integration/docusaurus/expectations/how_to_edit_an_expectation_suite get_suite"
+```
+
+Now use the `show_expectations_by_expectation_type()` to print the Suite to console or Jupyter Notebook.
+
+```python name="tests/integration/docusaurus/expectations/how_to_edit_an_expectation_suite show_suite"
+```
+
+Your output will look something similar to this: 
+
+```bash 
+[ { 'expect_column_values_to_be_between': { 'auto': True,
+                                            'column': 'passenger_count',
+                                            'domain': 'column',
+                                            'max_value': 6,
+                                            'min_value': 1,
+                                            'mostly': 1.0,
+                                            'strict_max': False,
+                                            'strict_min': False}},
+  { 'expect_column_values_to_not_be_null': { 'column': 'pickup_datetime',
+                                             'domain': 'column'}}]
+```
+
+### 5. Instantiate ExpectationConfiguration 
+
+From the Expectation Suite, you will be able to create an ExpecationConfiguration object using the output from `show_expectations_by_expectation_type()` Here is the example output of the first Expectation in our suite.
+
+It runs the `expect_column_values_to_be_between` Expectation on the `passenger_count` column and expects and max and min values to be `6` and `1` respectively. 
```

Comment:
Feel free to disregard if you don't agree, but min first feels more natural to me.
```suggestion
It runs the `expect_column_values_to_be_between` Expectation on the `passenger_count` column and expects min and max values to be `1` and `6` respectively. 
```

Reply:
great catch @anthonyburdi thank you